src/TransitorySensitivities.jl

```diff
@@ -149,52 +149,58 @@ end
 function normalizefulltransitorysensitivities!(dSdt,sim::Simulation,t)
     y = sim.sol(t)
     @views ns = y[sim.domain.indexes[1]:sim.domain.indexes[2]]
-    dSdt .*= sim.p'
+    #only fully normalize rate coefficient parameters
+    dSdt[:,sim.domain.parameterindexes[1]+length(sim.species):sim.domain.parameterindexes[2]] .*= sim.domain.p[sim.domain.parameterindexes[1]+length(sim.species):sim.domain.parameterindexes[2]]'
     @views dSdt[sim.domain.indexes[1]:sim.domain.indexes[2],:] ./= ns
-    return dSdt
+    return dSdt[vcat(Array(sim.domain.indexes[1]:sim.domain.indexes[2]),[i for (j,i) in enumerate(sim.domain.indexes) if j > 2]),sim.domain.parameterindexes[1]:sim.domain.parameterindexes[2]]
 end
 
 function normalizefulltransitorysensitivities!(dSdt,ssys::SystemSimulation,t)
     y = ssys.sol(t)
-    dSdt .*= ssys.p'
     for sim in ssys.sims
+        #only fully normalize rate coefficient parameters
+        @views dSdt[:,sim.domain.parameterindexes[1]+length(sim.species):sim.domain.parameterindexes[2]] .*= ssys.p[sim.domain.parameterindexes[1]+length(sim.species):sim.domain.parameterindexes[2]]'
         @views ns = y[sim.domain.indexes[1]:sim.domain.indexes[2]]
-        @views dSdt[sim.domain.indexes[1]:sim.domain.indexes[2],:] ./= ns
+        @views dSdt[vcat(Array(sim.domain.indexes[1]:sim.domain.indexes[2]),[i for (j,i) in enumerate(sim.domain.indexes) if j > 2]),:] ./= ns
```

Comment:
Is this right? Why do we normalize the thermochemical sensitivity by ns?

Reply:
The thermochemical sensitivity is initially dc/dHf298 that we convert to  dlog(c)/dHf298 by dividing by ns.

Reply:
I see. Yeah, that makes sense.

Reply:
Sorry help me understand this dSdt matrix, is it not `dSdt[i,j] = di / dj`?

Reply:
I think we only want to normalize the part of the species